packages/react/src/components/Button/Button-test.js

```diff
@@ -216,6 +216,17 @@ describe('Ghost Button', () => {
 
 describe('Small Button', () => {
   describe('Renders as expected', () => {
+    const wrapper = shallow(<Button size="small" className="extra-class" />);
+
+    it('Has the expected classes for small', () => {
```

Comment:
Small nit:

```suggestion
    it('has the expected classes for small', () => {
```

Reply:
@joshblack this is just copy-pasted from the other tests in the file, but i can edit them all if it makes them read better

Reply:
@lovemecomputer yeah, if you could that'd be awesome! This is obviously just a formatting thing though so not too important, but we should try and stay consistent (even though we contradict ourselves sometimes 😂)